packages/server/lib/cloud/protocol.ts

```diff
@@ -154,6 +154,10 @@ export class ProtocolManager implements ProtocolManagerShape {
     this.invokeSync('urlChanged', input)
   }
 
+  resetTest (testId: string): void {
```

Comment:
are we resetting the test or setting the test / current test? When I think reset I think of purging the previous value but this looks to actually set the current / following test from the previous. Is that right?

```suggestion
  resetTest (testId: string): void {
```